src/core/readyException.js

```diff
@@ -0,0 +1,13 @@
+define( [
+	"../core"
+], function( jQuery ) {
+
+"use strict";
+
+jQuery.readyException = function( error ) {
```

Comment:
Do we really need to define a default handler? I think just having the hook available is enough for consumer use cases.